mne/io/reference.py

```diff
@@ -123,6 +125,80 @@ def _apply_reference(inst, ref_from, ref_to=None, copy=True):
     return inst, ref_data
 
 
+def add_eeg_reference(inst, ref_channels, copy=True):
+    """Add reference channel(s) to data
```

Comment:
I should leave the plural because there may a case where you want to add more than one.